README.md

```diff
@@ -1,37 +1,38 @@
-# Privacy Policy for Topics
+Sure, below is your updated Privacy Policy including the new permissions:
 
-Last updated: 5/28/2023
+**Privacy Policy for Topics**
+**Last updated: 5/28/2023**
 
 This Privacy Policy describes how Paulo Briceno ("we", "us", or "our") collects, uses, and shares information in connection with your use of our mobile application Topics (the "App").
 
-## Information We Collect
+**Information We Collect**
 
-Camera: Our app requires access to your device's camera to enable Optical Character Recognition (OCR) functionalities and to handle images provided by the user. We use the images captured by the camera to provide these features and improve your user experience.
+**Camera:** Our app requires access to your device's camera to enable Optical Character Recognition (OCR) functionalities and to handle images provided by the user. We use the images captured by the camera to provide these features and improve your user experience.
 
-Storage: Our app requires access to your device's storage to store internal state related to multiple versions. This helps us maintain your preferences and provide a consistent user experience.
+**Storage:** Our app requires access to your device's storage to store internal state related to multiple versions. This helps us maintain your preferences and provide a consistent user experience.
 
-User Data: We store user data in Firebase, adhering to strict authentication policies to ensure the security and privacy of your data. This data is used to personalize your experience and provide you with relevant features and services.
+**Microphone and Internet:** Our app requires access to your device's microphone and internet connection to facilitate the voice and audio related functionalities. This data is not stored or transmitted and is used exclusively for providing the stated functionalities within the app.
 
-## How We Use Your Information
+**Bluetooth:** The app requires Bluetooth access for potential future features related to device communication and interaction. No data is extracted from your Bluetooth devices by our app.
```

Comment:
Why require permissions for a service that's not implement yet? Wouldn't it be better to wait unit something needs the Bluetooth connectivity to ask?

Reply:
https://github.com/pabrcno/topics/blob/main/android/app/src/main/AndroidManifest.xml

Utilizamos bt para los text to speech functionality, it is needed for headphone connection

Reply:
Then should that be mentioned here?